src/lib/colourType.js

```diff
@@ -12,4 +13,4 @@ export const isHex = colour => {
 
 export const isRGBorHSL = colour => /^(rgb|hsl)a?\(.*?\)/.test(colour);
 
-export const isKeyword = colour => has(colourNames, colour.toLowerCase());
+export const isKeyword = colour => has.call(colourNames, colour.toLowerCase());
```

Comment:
I also think it is not to much of a leap of faith to trust that people don't override `hasOwnProperty`, or even worse, mutate `Object.prototype`. So the code should really just be `colourNames.hasOwnProperty(colour.toLowerCase())`.

In this case we know that `colourNames` is a well behaved object, so it is only if someone else messes with the `Object.prototype` that this could fail. How many packages on npm do you think does that, I'm curious.